### PR TITLE
NETOBSERV-122: liveness probes for goflow-kube pod

### PR DESCRIPTION
**Pull request by mariomac**
Configures the `goflow-kube` deployment to use pod liveness probes.

This PR can't be merged until the related counterpart in goflow-kube is merged: https://github.com/netobserv/goflow2-kube-enricher/pull/19

### CODE REVIEW

File: config/crd/bases/flows.netobserv.io_flowcollectors.yaml [hunk 1]
```diff
@@ -107,6 +107,14 @@ spec:
               goflowkube:
                 description: GoflowKube contains settings related to goflow-kube
                 properties:
+                  healthPort:
```
Comment (OlivierCazade) [inline config/crd/bases/flows.netobserv.io_flowcollectors.yaml:110]: This changes should also be done to bundle/manifests/flows.netobserv.io_flowcollectors.yaml

**Review by OlivierCazade — APPROVED** [review]
LGTM, just a comment about crd definition, but may be we should create a test to ensure both files are equals.

File: config/crd/bases/flows.netobserv.io_flowcollectors.yaml (hunk 1 above)
Comment (mariomac) [reply config/crd/bases/flows.netobserv.io_flowcollectors.yaml:110]: Do we need to add it manually? Didn't find any makefile target that regenerated those manifests

Comment (mariomac) [reply config/crd/bases/flows.netobserv.io_flowcollectors.yaml:110]: Nevermind, I copied the additions manually.

**Pull request merged**